docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc

```diff
@@ -19,7 +19,7 @@ $ sudo cp -R /etc/ceph $HOME/ceph_client_backup
 $ sudo cephadm shell -- ceph mgr stat
 ----
 
-. Fail the `ceph-mgr` if it is active on the source node or target node:
+. Fail the `ceph-mgr` if it is active on the source or draining node :
```

Comment:
can we simply remove `or target node` here? I wouldn't add `or draining node`

Reply:
done